R/google_analytics.R

```diff
@@ -31,12 +44,28 @@ getGoogleAnalytics <- function(tableId, lastNDays = 30, dimensions, metrics, tok
   }
   token <- getGoogleTokenForAnalytics(tokenFileId)
 
-  start_date <- as.character(lubridate::today() - lubridate::days(lastNDays))
-  #end_date <- as.character(lubridate::today() - lubridate::days(1))
-  end_date <- as.character(lubridate::today())
+  if(dateRangeType == "lastNDays") {
+    if(is.null(lastN)) {
+      lastN <- lastNDays # For backward compatibility
```

Comment:
Please leave comment on detail of the backward compatibility this is about.

Reply:
Added commentl